README.md

```diff
@@ -62,6 +65,7 @@ This function receives a string input and converts it to mOcKiNgCaSe.
 | Param | Type | Default | Description |
 | --- | --- | --- | --- |
 | input | <code>string</code> |  | string to be converted |
+| input | <code>array</code>|  | array to be converted. Array must be an array of strings. __Example__: ['foo','bar'] is a valid array. ['foo',2] is not.
```

Comment:
For this project I use [jsdoc](http://usejsdoc.org/) to describe the function and after generate the markdown using [jsdoc2md](https://github.com/jsdoc2md/jsdoc-to-markdown). 
For the parameter that can be of 2 types description should be like 
```js
/**
* @param {(string|string[])} input - string or array of strings to be converted
*/
```
This generate
<code>string</code> \| <code>Array.&lt;string&gt;</code>
```md
<code>string</code> \| <code>Array.&lt;string&gt;</code>
```
So no need to create another line, just change current `| input | <code>string</code> |  | string to be converted |`